src/pages/Decks.spec.js

```diff
@@ -0,0 +1,28 @@
+import Decks from './Decks';
+
+import { render, screen } from '@testing-library/react';
+import { userEvent } from '@testing-library/user-event';
+import { MemoryRouter } from 'react-router-dom';
+
+describe('Decks', () => {
+  it('renders 6 buttons', () => {
+    render(
+      <MemoryRouter>
+        <Decks />
+      </MemoryRouter>
+    );
+    const allButtons = screen.getAllByRole('button');
```

Comment:
i think you can delete this, since you are more specifically testing that the buttons are there
```suggestion

```

Reply:
> i think you can delete this, since you are more specifically testing that the buttons are there

I kept it as an additional security, because if would add buttons and forget to specifically test them I would recognize it by the number of buttons 🙂